front/src/components/Admin/Users.vue

```diff
@@ -16,40 +16,43 @@
         </div>
       </div>
     </q-card-section>
-    <q-card-section>
+
+    <q-card-section class="q-pa-none">
       <q-table
         flat
-        bordered
+        dense
         :rows-per-page-options="[0]"
         :loading="loading"
         :columns="columns"
         :rows="users"
         hide-pagination
       >
         <template #body-cell-id="{ value }">
-          <q-td auto-width class="text-right">
+          <q-td auto-width>
             {{ value }}
           </q-td>
         </template>
         <template #body-cell-login="{ value }">
-          <q-td class="text-right">
+          <q-td>
             {{ value }}
           </q-td>
         </template>
```

Comment:
Personally I don't like that the username is all the way on the left. Also the distance between a username and the role is too big. In the previous UI I could easily see which role was for which user, now my eyes have to move a lot to match those values. I would propose to swap last active and role and make the display name next to the role again.

Reply:
I moved the role selector to the left (right to `id`). This way it also matches the Team page.